e2e-tests/IIoTPlatform-E2E-Tests/PublishSingleNodesTestTheory.cs

```diff
@@ -229,6 +231,19 @@ public class PublishSingleNodesTestTheory
 
         }
 
-        
+        [Fact, PriorityOrder(9)]
+        public async Task Test_WaitForModules_Expect_Success() {
```

Comment:
> Test_WaitForModules_Expect_Success()  [](start = 25, length = 38)

please update the branch, there is already a place holder for waiting defined

Reply:
Inserted this test content instead of that placeholder.